crates/trigger/src/stdio.rs

```diff
@@ -142,20 +161,37 @@ enum ComponentStdioWriterState {
 }
 
 impl ComponentStdioWriter {
-    pub fn new(log_path: &Path, follow: bool) -> anyhow::Result<Self> {
+    pub fn new(log_path: Option<&Path>, follow: bool) -> anyhow::Result<Self> {
+        match log_path {
+            Some(log_path) => Self::new_forward(log_path, follow),
+            None => Self::new_inherit(),
```

Comment:
Is it intentional that this does not consider `follow`?  I'm a bit fuzzy on what these two things are doing, but if `Inherit` is sending app logs to stdio then I feel like it should respect `follow`.

Reply:
Correct! Before we just called `inherit_stdio` on the store builder.

Reply:
Resolving this b/c follow is only relevant when we're trying to forward the content to a file